docs/components/DocsContent.vue

```diff
@@ -54,10 +54,11 @@
     display: inline-block;
     text-decoration: none;
     margin-right: 28px;
+    cursor: pointer;
   }
 
   .header__nav--item:hover {
-    font-weight: 600;
+    transform: scale(1.1);
```

Comment:
Hi,  I think ```font-weight: 600``` and ```transform: scale(1.1)``` have pretty much the same effect when hover a link. So there is no need to use ```transform scale``` to replace ```font-weight```. Or you have some others considerations ?

Reply:
Yes, you're right. There's no need to use a transform here. On https://developer.microsoft.com/en-us/fabric#/components/button, 'font-weight: 600' is also applied on hover, but the effect looks more "centered". I wonder how they achieve that effect.

Reply:
I know how to make the effect looks more "centered".

```css
.header__nav--item {
  display: inline-block;
  text-decoration: none;
  margin-right: 28px;
  cursor: pointer;
  text-align: center; /* need this one */
}
```